README.md

```diff
@@ -1,2 +1,58 @@
-# hellomicroverse
-In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+   <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> > In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+- Linters
```

Comment:
- [ ] Please as highlighted by the previous reviewer, kindly customize this section of README file. This will make a non-professional to understand what your project is all about. 👍